packages/SystemsThinkingWorkbench-Test.package/STWConnectionTest.class/instance/testSuccesfulPlacement.st

```diff
@@ -14,5 +14,4 @@ testSuccesfulPlacement
 	
 	connection connect: node1 with: node2.
 	
-	self assert: (connection vertices = {50@radius. 50@(50-radius)}).
-	
+	"TODO: Test bezier, but how?"
```

Comment:
choose a point that should be part of the curve, make a screenshot, print it, then scan it into gimp and look if the point is black.

no for real. cant you ask squak if a certain point is black or if there is a morph?